ai/ai-react-app/src/services/firebaseAIService.ts

```diff
@@ -0,0 +1,160 @@
+import { initializeApp, FirebaseApp } from "firebase/app";
+import {
+  HarmCategory,
+  HarmBlockThreshold,
+  Schema,
+  FunctionDeclaration,
+  Part,
+  Content,
+  CountTokensResponse,
+  GenerativeModel,
+  ModelParams,
+  ImagenModelParams,
+  FunctionCall,
+} from "firebase/vertexai";
```

Comment:
Missed a spot to update to "firebase/ai" I think

Reply:
oops- fixed!